accepted/0003-interactive-audit-resolver.md

```diff
@@ -0,0 +1,155 @@
+# Audit resolutions
+
+## Summary
+
+Historically this proposal was titled "Interactive audit resolver". That idea was too wide to introduce in one go and parts of it are better served in userland packages. This is now the core subset of the original proposal containing support for resolutions file.
+
+This proposal is for adding means for a human to resolve issues from `npm audit` by making and documenting decisions to ignore particular false positives. 
+The implementation should introduce support for reading and applying decisions from `audit-resolve.json` file
+
+Features:
+- Let users save tecisions about vulnerabilities and change `npm audit` behavior to accomodate that.
```

Comment:
Was "tecisions" supposed to say "decisions" ?

Reply:
That's definitely one of the possibilities.